src/server/utilities/mvtHeader/enabledExperimentsList.js

```diff
@@ -0,0 +1,14 @@
+const enabledExperimentList = [
+  {
+    name: 'simorgh_dark_mode',
+    services: ['pidgin', 'mundo'],
+    pageTypes: ['STY', 'IDX'],
+  },
+  {
+    name: 'full_stack_test',
+    services: ['pidgin'],
+    pageTypes: ['STY', 'IDX'],
```

Comment:
There is a potential issue with this approach in that if a page doesn't apply the experiment in every code path (e.g. an experiment with ads where only certain assets have them enabled) we will vary unecessarily making our cache less effective downstream of us.

I had an idea to [update a React Context](https://reactjs.org/docs/context.html#updating-context-from-a-nested-component) wrapped around our pages when we actually apply a variation to account for any more fine grained logic around where an experiment is applied. We could then reference this value in the server to determine whether or not to set the vary header.

I think we could then avoid a centralised experiment list as we could maintain the logic close to the place in code where we apply the variation.